c++/src/kj/compat/tls-test.c++

```diff
@@ -28,7 +28,7 @@
 #include <openssl/opensslv.h>
 
 #if _WIN32
-#define WIN32_LEAN_AND_MEAN
+#include "win32-api-version.h"
```

Comment:
This one needs to be in `<>` -- this is breaking the CI build:

D:\a\capnproto\capnproto\c++\src\kj\compat\tls-test.c++(31,10): fatal error C1083: Cannot open include file: 'win32-api-version.h': No such file or directory [D:\a\capnproto\capnproto\build-output\src\kj\kj-heavy-tests.vcxproj]